frontend/server/src/Controllers/Problem.php

```diff
@@ -57,35 +57,39 @@ class Problem extends \OmegaUp\Controllers\Controller {
     /**
      * Validates a Create or Update Problem API request
      *
-     * @param \OmegaUp\Request $r
+     * @return array{languages: string, problem: \OmegaUp\DAO\VO\Problems|null, selectedTags: array{tagname: string, public: bool}[]|null}
```

Comment:
```suggestion
     * @return array{languages: string, problem: \OmegaUp\DAO\VO\Problems|null, selectedTags: array{tagname: string, public: bool}[]}
```

aquí `selectedTags` nunca es nulo.

Reply:
Al parecer, si. El problema es que estaba declarando incorrectamente lo que obtenía del `json_decode`. 

Ahora tengo otro problema: creé la clase Tag, pero no sé como puedo mandar los datos del `json_decode` al objeto. Intenté con: `$tag = new \OmegaUp\Tag($tag->tagname, $tag->public);` pero me sigue mandando: `Undefined property: stdClass::$tagname`

Reply:
eso no debería ser `new \OmegaUp\DAO\VO\Tags($tag['tagname'])`? te dejé otro comentario abajo para no convertir el resultado de `json_decode` en un `stdClass`, sino un arreglo asociativo.

Reply:
`\OmegaUp\DAO\VO\Tags` No tiene el atributo `public`, entonces perdería esa información al guardarlo en un problema. Parece que con el cambio de obtener arreglo asociativo se solucionan los errores